jobs/ci-kubernetes-e2e-gce-scalability.env

```diff
@@ -13,6 +13,8 @@ NUM_NODES=100
 ALLOWED_NOTREADY_NODES=1
 REGISTER_MASTER=true
 
+# Switch off image puller to workaround #44701
```

Comment:
Let's not touch previous releases (1.5, 1.6), but let's fix for all current ones.